lib/ui/window.dart

```diff
@@ -116,9 +116,10 @@ class WindowPadding {
   }
 }
 
-/// An identifier used to select a user's language and formatting preferences,
-/// consisting of a language and a country. This is a subset of locale
-/// identifiers as defined by BCP 47.
+/// An identifier used to select a user's language and formatting preferences.
+/// This represents a [Unicode Language
```

Comment:
nit: put the first sentence on its own in a separate paragraph at the top. The first paragraph is used for summaries and autocomplete help and so on.